adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenSilentHandlerTest.java

```diff
@@ -46,10 +49,17 @@
 import javax.crypto.spec.PBEKeySpec;
 import javax.crypto.spec.SecretKeySpec;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
it's used around line 343, it's just not listed in the diff. So it is with those below it.